### PR TITLE
Feat/add template cat plumbing

### PR DESCRIPTION
**Pull request by andrewleith**
# Summary | Résumé

This PR adds some of the ground work to get template categories working in ADMIN:
- 2 new admin routes to view the list of categories as well as an individual category (no create/update/delete yet)
- The skeleton for a new api client for template categories
- A work-in-progress banner since these admin pages could go live before we are totally done

## Template categories list page
image

## Template category page 
image

# Related cards
- https://github.com/cds-snc/notification-planning/issues/1588

# Related PRs
- https://github.com/cds-snc/notification-terraform/pull/1382

### CODE REVIEW

File: app/notify_client/template_category_api_client.py
Comment (andrewleith) [inline app/notify_client/template_category_api_client.py:1]: Skeleton template category API.  Will be fully fleshed out when the DB work is complete.

File: app/main/forms.py [hunk 1]
```diff
@@ -1829,3 +1829,27 @@ class BrandingRequestForm(StripWhitespaceForm):
...
+class TemplateCategoryForm(StripWhitespaceForm):
+    name_en = StringField("Name EN", validators=[DataRequired(message="You need to upload a file to submit")])
+    name_fr = StringField("Name FR", validators=[DataRequired(message="You need to upload a file to submit")])
+    desc_en = StringField("Desc EN", validators=[DataRequired(message="You need to upload a file to submit")])
+    desc_fr = StringField("Desc FR", validators=[DataRequired(message="You need to upload a file to submit")])
+    hidden = RadioField("Hidden", choices=[("True", "Yes"), ("False", "No")])
+    email_priority = RadioField(
+        "Email Priority",
+        choices=[
+            ("bulk", "Bulk"),
```
Comment (jzbahrai) [inline app/main/forms.py:1843]: lets change this to "High" , "Medium", "Low" - we renamed our queues to that, and going forward want to use this everywhere

File: app/main/forms.py [hunk 2]
```diff
@@ -1829,3 +1829,27 @@ class BrandingRequestForm(StripWhitespaceForm):
...
+            ("normal", "Normal"),
+            ("normal", "High"),
+        ],
+    )
+    sms_priority = RadioField(
+        "SMS Priority",
+        choices=[
+            ("bulk", "Bulk"),
```
Comment (jzbahrai) [inline app/main/forms.py:1851]: same

**Review by jzbahrai — APPROVED** [review]
added a comment, else lgtm!

File: app/main/forms.py (hunk 1 above)
Comment (andrewleith) [reply app/main/forms.py:1843]: Ah good to know, will update!

File: tests_cypress/cypress.config.js [hunk 3]
```diff
@@ -52,6 +52,6 @@ module.exports = defineConfig({
     viewportWidth: 1280,
     viewportHeight: 850,
     testIsolation: true,
-    // retries: 3
+    retries: 3
```
Comment (andrewleith) [inline tests_cypress/cypress.config.js:55]: This was mistakenly disabled in a different PR, adding it back in.

**Pull request merged**